examples/src/pages/Simple.tsx

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from "three";
+import CameraControls from "camera-controls";
+import {
```

Comment:
This file is probably the first one that should be "Reactified". I would like seeing more reactive code (splitting fetch model and setup renderloop etc).

Meaning for instant one has a:
```
useEffect( () => {
  // fetch new model
  // then change camera
  // request rerender?
}, [modelRevision, modelUrl])
```

Along these lines?